source/fundamentals/connection/tls.txt

```diff
@@ -152,6 +155,36 @@ please refer to the `JSSE Reference Guide
 
 .. _tls-disable-hostname-verification:
 
+Configure a Client-Specific Trust Store and Key Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a client-specific trust store and key store using the
+``init()`` method of the ``SSLContext`` class.
+
+The following code snippet configures a client-specific trust store and key
+store:
+
+.. code-block:: java
+
+   String algorithm = "TLS";
+   try {
+      final SSLContext sslContext = SSLContext.getInstance(algorithm);
+      sslContext.init(<Your Key Managers>, <Your Trust Managers>, <A Secure Random Number Generator>);
+      MongoClientSettings settings = MongoClientSettings.builder()
+               .applyToSslSettings(builder ->
+                     builder.enabled(true)
+                              .context(sslContext)
+                              .build())
+               .build();
```

Comment:
Question: Did you mean to have `.build()` twice?

Reply:
Great question. That was intentional, but I think it might be incorrect now that I am [double checking the API documentation](https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#applyToSslSettings(com.mongodb.Block)). Fix: